snap/snapcraft.yaml

```diff
@@ -48,7 +53,9 @@ parts:
       - --prefix=/
     build-packages:
       - jq
+    build-environment:
+      - PKG_CONFIG_PATH: ${PKG_CONFIG_PATH:+$PKG_CONFIG_PATH:}$CRAFT_STAGE/usr/lib/$CRAFT_ARCH_TRIPLET/pkgconfig:$CRAFT_STAGE/usr/lib/pkgconfig
```

Comment:
Shouldn't this be done already in the gnome extension?

Reply:
It should, but if I remove that...

![imagen](https://user-images.githubusercontent.com/105285003/192968761-c53740d2-257d-4c5d-a00b-a95bd3cd63a0.png)

I'll check the gnome extension, anyway.

Reply:
Yes. It was some garbage due to the change of the pkgconfig name for glib.

Reply:
Fixed